### PR TITLE
add Cantonese

### PR DESCRIPTION
**Pull request by irvin**
add Cantonese (yue), fix https://github.com/common-voice/sentence-collector/issues/396

### CODE REVIEW

File: server/lib/languages.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const ADDITIONAL_LANGUAGES = [
   {
     id: 'bas',
     name: 'Basaa',
-    nativeName: 'Ɓàsàa',
+    nativeName: 'Ɓàsàa',
```
Comment (irvin) [inline server/lib/languages.js:37]: I had no idea why there is a diff @@

Comment (MichaelKohler) [reply server/lib/languages.js:37]:
```suggest
    nativeName: 'Ɓàsàa',
```

Comment (MichaelKohler) [inline server/lib/languages.js:37]:
```suggestion
    nativeName: 'Ɓàsàa',
```

**Review by MichaelKohler — CHANGES_REQUESTED** [review]

**Review by MichaelKohler — APPROVED** [review]
Thanks!

**Pull request merged**